assets/js/components/Spinner.stories.jsx

```diff
@@ -0,0 +1,28 @@
+import Spinner from './Spinner';
+
+export default {
+  title: 'Spinner',
+  component: Spinner,
+};
+
+export const Default = {
+  argTypes: {
+    className: {
+      control: 'text',
+      description: 'Add padding or margin',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '""' },
```

Comment:
what happens with `{ summary: '' },`?

Reply:
Ah well done, nice catch. This is a left over! 
Storybook detects the default value. No need to add it here.